libraries/rush-lib/src/logic/pnpm/PnpmOptionsConfiguration.ts

```diff
@@ -311,6 +329,15 @@ export class PnpmOptionsConfiguration extends PackageManagerOptionsConfiguration
 
   public readonly jsonFilename: string | undefined;
 
+  /**
+   * The `pnpmLockfilePolicies` setting defines the policies that govern the `pnpm-lock.yaml` file.
+   *
+   * @remarks
+   * Available options:
+   *  - disallowInsecureSha1: Forbid sha1 hashes in `pnpm-lock.yaml`
```

Comment:
It seems like this comment is likely to get outdated when people add things to `IPnpmLockfilePolicies`.